scripts/alignmentfile_pairedness.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import pysam
+import argparse
+import pathlib
+
+def parse_command() -> argparse.Namespace:
+    """Function to parse the command line input
+    Args:
+        None
+    Return:
+        Namespace: returns the args as a standard Namespace object
+    """
+    parser = argparse.ArgumentParser(description='Determine if input BAM,CRAM,SAM file is paried or single end')
+    parser.add_argument('--input_reads',
+        required=True,
+        help="Path to the BAM,CRAM,SAM file. If providing a CRAM, you must also provide an input_reference.")
+    parser.add_argument('--input_reference',
+        help="For CRAM only, provide the reference file used when making the input_reads")
+    parser.add_argument('--max_reads',
+        default=200_000,
```

Comment:
What is that syntax for a number? `_` ?

Reply:
Just a way of representing numbers in python. See https://peps.python.org/pep-0515/